content/en/about/community/customers/index.md

```diff
@@ -10,6 +10,7 @@ skip_seealso: true
 Here are just a few of the organizations we know are using Istio. If you want to edit this page, [please send a pull request!](https://github.com/istio/istio.io/tree/master/content/en/about/community/customers)
 
 <div class="logo-gallery">
+    {{< company_logo link="https://www.aihuishu.com" logo="./aihuishou.jpeg" alt="Aihuishou" >}}
```

Comment:
Link returns not found for me.Suspect it's missing a `o` and should be https://www.aihuishou.com.

Reply:
Oh my God